how-to/containerd-kata.md

```diff
@@ -185,14 +185,24 @@ The following configuration includes three runtime classes:
            SystemdCgroup = false
 ```
 
+From Containerd v1.2.4 and Kata v1.6.0, there is a new runtime option supported, by which you can specify a specific 
+kata configure file as below:
+
+```toml
+      [plugins.cri.containerd.runtimes.kata]
+         runtime_type = "io.containerd.kata.v2"
+	 [plugins.cri.containerd.runtimes.kata.options]
+	   ConfigPath = "/etc/kata-containers/config.toml"
+```
+
+This "ConfigPath" option is optional, if you doesn't specify it, shimv2 will first try to get the configure file from the environment
```

Comment:
Lines 198-200 suggested rewrite: 
This “ConfigPath” option is optional. If you do not specify it, shimv2 first tries to get the configure file from the environment variable “KATA_CONF_FILE”. If it is not set in that variable, then shimv2 searches the configuration file in the default paths, similar to ...”